common/test/acceptance/pages/lms/instructor_dashboard.py

```diff
@@ -1415,6 +1415,13 @@ def click_invalidate_certificate_button(self):
         self.get_selector('#invalidate-certificate').click()
 
     @property
+    def self_generation_certificate_enabled_button(self):
```

Comment:
Please try finding a more appropriate name for this method. Maybe like `self_generate_certificate_enabled_button `.